### PR TITLE
fix dtype issue

### PR DESCRIPTION
**Pull request by ssuttles-usgs**
Cast global attribute variable "dir_offset" and "magnetic_variation" as floats when used in wind direction calculations in wxt.py. Remove commented out lines. Ran tests test_scripts.py and processed test dataset.

### CODE REVIEW

**Comment by ssuttles-usgs** [comment]
Reformatted wxt.py using black.

File: stglib/wxt.py [hunk 1]
```diff
@@ -84,21 +84,34 @@ def cdf_to_nc(cdf_filename):
...
+    if "WD_410" in ds:
+        ds["WD_410"] = ds["WD_410"].astype(float)
+    if "WD_gust" in ds:
+        ds["WD_gust"] = ds["WD_gust"].astype(float)
 
     # If sensor wasn't pointing to magnetic north, apply offset to direction
     if "dir_offset" in ds.attrs:
         if "WD_min" in ds:
-            ds["WD_min"].values = ds["WD_min"].values + ds.attrs["dir_offset"]
+            ds["WD_min"].values = ds["WD_min"].values + ds.attrs["dir_offset"].astype(
```
Comment (dnowacki-usgs) [inline stglib/wxt.py:98]: I am pretty sure all these `.values` are unnecessary. Have you tried without? Mostly a readability thing

**Comment by dnowacki-usgs** [comment]
If you have the bandwidth to remove and test without, I think we should remove them. Occasionally we need to use `.values` to pass to some functions which can't accept an Xarray object and instead require a numpy array or similar. But otherwise, if the function can work without it, it just makes things a bit cleaner. Thanks.

File: stglib/wxt.py (hunk 1 above)
Comment (ssuttles-usgs) [reply stglib/wxt.py]: Should be all set now. removed .values and ran test on 2 datasets and test_scripts.py. all passed.

**Pull request merged**